docs-src/concepts.md

```diff
@@ -104,6 +104,13 @@ Route objects define protocol-specific rules for mapping requests from a Gateway
 `HTTPRoute` and `TCPRoute` are currently the only defined Route objects. Additional protocol-specific Route
 objects may be added in the future.
 
+### BackendPolicy
+
+BackendPolicy provides a way to configure backends. For the purpose of this API,
+a backend is any resource that a route can forward traffic to. A common example
+of a backend is a Service. Configuration at this level is currently limited to
+TLS, but will expand in the future to support more advanced policies.
```

Comment:
Rob, I think it is important to include what can go into BackendPolicy. What I mean to say is, we should communicate that configuration that relates to the Gateway-Upstream hop should go in here. Linking #196 here might be a good idea as well.

Reply:
Good call - updated to make this more clear.